<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главная</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tw_Main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="searchTab">
       <attribute name="title">
        <string>Поиск</string>
       </attribute>
       <layout class="QVBoxLayout" name="vLayout_price">
        <item>
         <layout class="QGridLayout" name="gBox_priceMain">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_searchName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Наименование</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbl_searchP1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Наим</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="le_searchP1"/>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btn_search">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lbl_searchP2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Доз</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="le_searchP2"/>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="btn_clearNames">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lbl_searchP3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Номер</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="le_searchP3"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hBox_priceBtns">
          <item>
           <widget class="QPushButton" name="btn_loadPrices">
            <property name="text">
             <string>Загрузить прайсы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clearMemory">
            <property name="text">
             <string>Очистить память</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_order">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Заявка</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="vSpacer1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>315</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QListWidget" name="lW_logger"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Настройки</string>
       </attribute>
       <layout class="QVBoxLayout" name="vLayout_settings">
        <item>
         <layout class="QHBoxLayout" name="hBox_settingsPrice">
          <item>
           <widget class="QLabel" name="lbl_pricesDir">
            <property name="text">
             <string>Прайсы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_pricesDir">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_openPricesDir">
            <property name="text">
             <string>Открыть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_editPricesDir">
            <property name="text">
             <string>Изменить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hBox_settingsReq">
          <item>
           <widget class="QLabel" name="lbl_requestDir">
            <property name="text">
             <string>Заявки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_requestDir">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_openRequestDir">
            <property name="text">
             <string>Открыть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_editRequestDir">
            <property name="text">
             <string>Изменить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tvw_settings">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hBox_settingsBtns">
          <item>
           <widget class="QPushButton" name="btn_addVendor">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_removeVendor">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_refresh">
            <property name="text">
             <string>Обновить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_setUp">
            <property name="text">
             <string>Настроить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_overwriteSettings">
            <property name="text">
             <string>Перезаписать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>158</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
